core/wiki/allfields.tid

```diff
@@ -1,13 +1,41 @@
 title: $:/snippets/allfields
 
+\whitespace trim
+\import [[$:/core/ui/ControlPanel/Basics]]
+
 \define renderfield(title)
-<tr class="tc-view-field"><td class="tc-view-field-name">''<$text text=<<__title__>>/>'':</td><td class="tc-view-field-value">//{{$:/language/Docs/Fields/$title$}}//</td></tr>
+<tr class="tc-view-field">
+	<td class="tc-view-field-name">
+		''<$text text=<<__title__>>/>'':
+	</td>
+	<td class="tc-view-field-value">
+		//<$text text={{$:/language/Docs/Fields/$title$}}/>//
+	</td>
+	<td class="tc-view-field-links">
+		<<show-filter-count "[has[$title$]sort[]]">>
+	</td>
+</tr>
 \end
-\whitespace trim
+
+\define renderfieldHeader()
```

Comment:
Would we be better of making these procedures?

Reply:
I did use procedures, but it breaks the "show-filter-count" macro, which is imported from [[$:/core/ui/ControlPanel/Basics]]. It expects the parameter as a `$title$`. 

I did not want to change all the stuff and make it backwards compatible, just to add this one button. Filters usually have to be used very different if we remove the "old substitution" style. 

That's why I used `\define`

Reply:
I think I can make the header a procedure. But IMO `renderfield()` should stay a macro

Reply:
I did change the macros to use procedures